crates/apollo-compiler/README.md

```diff
@@ -53,39 +55,19 @@ Older version may or may not be compatible.
 You can get started with `apollo-compiler`:
 ```rust
 use apollo_compiler::Schema;
+use apollo_compiler::ExecutableDocument;
 
-let input = r#"
-  interface Pet {
-    name: String
-  }
-
-  type Dog implements Pet {
-    name: String
-    nickname: String
-    barkVolume: Int
-  }
-
-  type Cat implements Pet {
-    name: String
-    nickname: String
-    meowVolume: Int
-  }
-
-  union CatOrDog = Cat | Dog
-
-  type Human {
-    name: String
-    pets: [Pet]
-  }
-
+let sdl = r#"
```

Comment:
Making examples shorter so there’s less to scroll on the docs landing page before getting to Rust items navigation